client/src/main/kotlin/no/nav/su/se/bakover/client/journalpost/HentFerdigstiltJournalpost.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.su.se.bakover.client.journalpost
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import no.nav.su.se.bakover.domain.Saksnummer
+import no.nav.su.se.bakover.domain.journalpost.FerdigstiltJournalpost
+import no.nav.su.se.bakover.domain.journalpost.JournalpostStatus
+import no.nav.su.se.bakover.domain.journalpost.JournalpostTema
+import no.nav.su.se.bakover.domain.journalpost.JournalpostType
+
+internal fun Journalpost?.toFerdigstiltJournalpost(saksnummer: Saksnummer): Either<JournalpostErIkkeFerdigstilt, FerdigstiltJournalpost> {
+    if (this == null) {
+        return JournalpostErIkkeFerdigstilt.FantIkkeJournalpost.left()
+    }
+    if (tema == null || tema != JournalpostTema.SUP.toString()) {
+        return JournalpostErIkkeFerdigstilt.JournalpostTemaErIkkeSUP.left()
+    }
+    if (journalposttype == null || journalposttype != JournalpostType.INNKOMMENDE_DOKUMENT.value) {
```

Comment:
Vi burde kanskje hatt med denne begrensninga i kdoc-en

Reply:
Litt usikker på hvilken dok du tenker på?

Reply:
Heter visst javadoc for *.java og kdoc for *.kt/kts. Mente at vi kunne legge på en kdoc på funksjonen for å si noe om forventning til konsumenten. Det er kanskje en snodig begresning at det kun gjelder innkommende dokumenter